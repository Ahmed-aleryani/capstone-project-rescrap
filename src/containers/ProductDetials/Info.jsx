import React from 'react';
import SubTitle from '../../components/SubTitle/SubTitle';

const Info = (props) => {
  return (
    <div className="mx-10 flex flex-col justify-center ">
      <div className="flex flex-wrap justify-center px-10">
        <div className="w-full md:w-1/2 lg:w-1/2">
          <SubTitle
            text="Details"
            classes="  uppercase font-semibold leading-5  tracking-widest  text-left  my-2  "
          />

          <div className="flex flex-col w-3/5 text-gray-700">
            <div className="flex flex-row justify-between border-b-2 border-gray-400 my-4">
              <SubTitle text="category" /> <SubTitle text={props.category} />
            </div>
            <div className="flex flex-row justify-between border-b-2 border-gray-400 my-4">
              <SubTitle text="Quantity" /> <SubTitle text={props.quantity} />
            </div>
            <div className="flex flex-row justify-between border-b-2 border-gray-400 my-4">
              <SubTitle text="Location" />
              <SubTitle text={props.addressDetails} />
            </div>
          </div>
        </div>
        <div className="w-full md:w-1/2 lg:w-1/2">
          <SubTitle
            text="Sellers"
            classes="  uppercase font-semibold my-2  "
          />
          <div className="flex flex-col w-2/3 text-gray-700 ">
            <div className="flex flex-row justify-between my-4 ">
              <div className="text-green-100">
                <i className="fa fa-user text-2xl" />
              </div>
              <h3 className="text-green-100 text-xl mx-4">{props.loggedIn?props.owner.name:"Login to see owner info"}</h3>
            </div>
            <div className="flex flex-row justify-between my-4 ">
              <div className="text-green-100">
                <i className="fa fa-at  text-2xl" />
              </div>
              <h3 className="border-b-2 border-gray-300 text-xl mx-4 ">
                
              {props.loggedIn?props.owner.email:"Login to see owner info"}
              </h3>
            </div>
            <div className="flex flex-row justify-between my-4  ">
              <div className="text-green-100">
                <i className="fas fa-phone-alt text-2xl" />
              </div>
              <h3 className="text-xl ">  {props.loggedIn?props.owner.phoneNumber:"Login to see owner info"}</h3>
            </div>
            {/* <div className="flex flex-row justify-between my-2 ">
              <div className="text-green-100">
                <i class="fas fa-globe-americas text-2xl" />
              </div>
              <h3 className="text-xl">{props.product.addressDetails}</h3>
            </div> */}
          </div>
        </div>
      </div>
    </div>
  );
};
export default Info;
