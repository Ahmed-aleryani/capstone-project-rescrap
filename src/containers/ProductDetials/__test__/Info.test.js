import React from 'react';
import renderer from 'react-test-renderer';
import Info from '../Info';

it('matches snapshot', () => {
  const x= renderer.create(<Info />).toJSON();
  expect(x).toMatchSnapshot();
});