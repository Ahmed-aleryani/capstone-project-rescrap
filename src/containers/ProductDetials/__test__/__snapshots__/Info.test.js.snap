// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="mx-10 flex flex-col justify-center "
>
  <div
    className="flex flex-wrap justify-center px-10"
  >
    <div
      className="w-full md:w-1/2 lg:w-1/2"
    >
      <h2
        className="text-2xl thin  uppercase font-semibold leading-5  tracking-widest  text-left  my-2  "
      >
        Details
      </h2>
      <div
        className="flex flex-col w-3/5 text-gray-700"
      >
        <div
          className="flex flex-row justify-between border-b-2 border-gray-400 my-4"
        >
          <h2
            className="text-2xl thinundefined"
          >
            category
          </h2>
           
          <h2
            className="text-2xl thinundefined"
          />
        </div>
        <div
          className="flex flex-row justify-between border-b-2 border-gray-400 my-4"
        >
          <h2
            className="text-2xl thinundefined"
          >
            Quantity
          </h2>
           
          <h2
            className="text-2xl thinundefined"
          />
        </div>
        <div
          className="flex flex-row justify-between border-b-2 border-gray-400 my-4"
        >
          <h2
            className="text-2xl thinundefined"
          >
            Location
          </h2>
          <h2
            className="text-2xl thinundefined"
          />
        </div>
      </div>
    </div>
    <div
      className="w-full md:w-1/2 lg:w-1/2"
    >
      <h2
        className="text-2xl thin  uppercase font-semibold my-2  "
      >
        Sellers
      </h2>
      <div
        className="flex flex-col w-2/3 text-gray-700 "
      >
        <div
          className="flex flex-row justify-between my-4 "
        >
          <div
            className="text-green-100"
          >
            <i
              className="fa fa-user text-2xl"
            />
          </div>
          <h3
            className="text-green-100 text-xl mx-4"
          >
            Login to see owner info
          </h3>
        </div>
        <div
          className="flex flex-row justify-between my-4 "
        >
          <div
            className="text-green-100"
          >
            <i
              className="fa fa-at  text-2xl"
            />
          </div>
          <h3
            className="border-b-2 border-gray-300 text-xl mx-4 "
          >
            Login to see owner info
          </h3>
        </div>
        <div
          className="flex flex-row justify-between my-4  "
        >
          <div
            className="text-green-100"
          >
            <i
              className="fas fa-phone-alt text-2xl"
            />
          </div>
          <h3
            className="text-xl "
          >
              
            Login to see owner info
          </h3>
        </div>
      </div>
    </div>
  </div>
</div>
`;
